source/administration/mmctl-cli-tool.rst

```diff
@@ -0,0 +1,2271 @@
+mmctl Command Line Tool (Beta)
+==============================
+
+The mmctl tool is a remote CLI tool for Mattermost which is installed locally and uses the Mattermost API. Authentication
+is done with either login credentials or an authentication token.
+
+Being installed locally allows a System Admin to run CLI commands even in instances where there is no access to the
+server (e.g., via SSH). This tool is currently in beta and can be used alongside the Mattermost CLI tool.
+In the future, the Mattermost CLI tool will be deprecated.
+
+**Notes**
+
+-  Parameters in CLI commands are order-specific.
+-  If special characters (``!``, ``|``, ``(``, ``)``, ``\``, ``'``, and ``"``) are used, the entire argument needs to be surrounded by single quotes (e.g. ``-password 'mypassword!'``, or the individual characters need to be escaped out (e.g. `` password mypassword\!``).
+- Team name and channel name refer to the handles, not the display names. So in the URL ``https://community.mattermost.com/core/channels/town-square`` team name would be ``core`` and channel name would be ``town-square``.
+
+
+**Options**
+
+.. code-block:: sh
+
+       --format string    the format of the command output [plain, json] (default "plain")
+       -h, --help         help for mmctl
+
+**Commands**
```

Comment:
Missing 'mmctl auth'.

Reply:
'version' and 'webhook' can be removed if we decide to remove from this doc for now.